share/spack/bash/spack-completion.in

```diff
@@ -304,6 +304,14 @@ _pretty_print() {
 
 complete -o bashdefault -o default -F _bash_completion_spack spack
 
+# Alias and completion for spacktivate
+alias spacktivate="spack env activate"
```

Comment:
I think the alias should be set in `setup-env.sh` so it works for all shells, not just Bash. The completion stuff is fine to remain here.